lib/utils/log/slog_handler_test.go

```diff
@@ -221,3 +223,101 @@ func TestSlogJSONHandlerReservedKeysOverrideTypeDoesntPanic(t *testing.T) {
 	require.Contains(t, logRecordMap, "msg")
 	require.InEpsilon(t, float64(123), logRecordMap["msg"], float64(0))
 }
+
+func TestSlogTextHandlerComponentPadding(t *testing.T) {
+	tests := []struct {
+		name      string
+		component string
+		padding   int
+		want      string
+	}{
+		{name: "padded component",
+			component: "foo",
+			padding:   11,
+			want:      "[FOO]       bar\n",
+		},
+		{name: "truncated component",
+			component: "foobarbazquux",
+			padding:   11,
+			want:      "[FOOBARBAZ] bar\n",
+		},
+		{name: "no padding",
+			component: "foobarbazquux",
+			padding:   0,
+			want:      "[FOOBARBAZQUUX] bar\n",
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			var buf bytes.Buffer
+
+			h := NewSlogTextHandler(&buf, SlogTextHandlerConfig{
+				Level:            slog.LevelDebug,
+				Padding:          tt.padding,
+				ConfiguredFields: []string{ComponentField},
+			})
+			// Override defaults set by NewSlogTextHandler.
+			if tt.padding == 0 {
+				h.cfg.Padding = 0
```

Comment:
`NewSlogTextHandler` sets this padding by default, but in the subsequent PRs I'm adding constructors for os_log and Event Logger that do not set the default. However, they both set a writer for a specific platform, so I hope changing this private value here is okay.